public/home.html

```diff
@@ -102,9 +102,17 @@
     <!-- list of documents -->
     <div v-if="mode==='unassigned' || mode === 'mytickets' || mode === 'search'" class="container">
       <div class="row">
-        <div class="col-md-12">
+        <div class="col-md-10">
           <h2>{{ mode }} tickets <span class="badge" v-if="numDocs">{{ docs.length }}</span></h2>
         </div>
+        <div class="col-md-2">
+          <h2>
+            <button v-if="showNotes && mode === 'unassigned'" v-on:click="showNotes=false;unAssignedTickets" class="btn btn-primary align-bottom">Hide Notes</button>
```

Comment:
Nice. I was wondering if the button text could be done with a ternary operator but didn't find that quickly.